pkg/cmd/roachtest/pgjdbc_blacklist.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package main
+
+var pgjdbcBlacklists = blacklistsForVersion{
+	{"v2.0", "pgjdbcBlackList2_0", pfjdbcBlackList2_0, "", nil},
+	{"v2.1", "pgjdbcBlackList2_1", pfjdbcBlackList2_1, "", nil},
+	{"v2.2", "pgjdbcBlackList19_1", pfjdbcBlackList19_1, "", nil},
+	{"v19.1", "pfjdbcBlackList19_1", pfjdbcBlackList19_1, "", nil},
+	{"v19.2", "pfjdbcBlackList19_2", pfjdbcBlackList19_2, "", nil},
+}
+
+// Please keep these lists alphabetized for easy diffing.
+// After a failed run, an updated version of this blacklist should be available
+// in the test log.
+var pfjdbcBlackList19_2 = blacklist{
```

Comment:
I think this should be pgjdbc, not pfjdbc

Reply:
Yeah, that seems right. Thanks for the catch! :D